data/Templates/Ccda/Resource/_AllergyIntolerance.liquid

```diff
@@ -42,15 +42,13 @@
                     ],
                     "onset": "{{ r.observation.effectiveTime.low.value | format_as_date_time }}", 
                 {% endif -%}
-                {% for templateId in r.observation.templateId -%}
-                    {% if templateId.root == "2.16.840.1.113883.10.20.22.4.8" -%}
-                        {% if r.observation.value.translation -%}
-                            "severity":"{{ r.observation.value.translation.displayName | downcase | get_property: 'ValueSet/AllergySeverity' }}",
-                        {% else -%}
-                            "severity":"{{ r.observation.value.displayName | downcase | get_property: 'ValueSet/AllergySeverity' }}",
-                        {% endif -%}
+                {% if r.observation.templateId | to_json_string | contains: '"2.16.840.1.113883.10.20.22.4.8"' -%}
```

Comment:
Maybe need to alternate because of the 'if' behavior we discussed today.

Reply:
Will fix all filters in ```if``` tag in another PR, now currently keep the condition check with others